app/scss/book-detail.scss

```diff
@@ -0,0 +1,41 @@
+.book-detail {
+  display: flex;
+}
+
+.book-detail-image {
+  width: 191px;
```

Comment:
why not 190px?

Reply:
In the styleguide it has 191px.
![image](https://user-images.githubusercontent.com/2124740/50846308-d12f1500-133c-11e9-8edb-1fd5cbbd3a5b.png)